Lib9c/Action/DPoS/Control/AllocateReward.cs

```diff
@@ -98,7 +98,7 @@ public static Address RewardAddress(Address holderAddress)
 
             FungibleAssetValue votePowerNumerator
                 = votes.Aggregate(
-                    Asset.ConsensusToken * 0, (total, next)
+                    Asset.ConsensusFromGovernance(0), (total, next)
```

Comment:
Are these kind of changes needed?
Why zero FAVs has been changed with conversion method?

Reply:
If there was a property like ConsensusToken.Zero, I think that would be the best way to go for this code, but since there isn't, I used it to be more consistent.

Reply:
https://github.com/planetarium/lib9c/pull/2521/commits/81a7df4007d464a987bd0e46ae6a4b349f72b6da
Reverted code to `Asset.ConsensusToken * 0`